pkg/db/migration/000002_create_knowledge_base_table.up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE knowledge_base (
+    id SERIAL PRIMARY KEY,
```

Comment:
Hi @Yougigun 
The convention in our current codebase is to use `uuid` as the primary key. I'd suggest that we continue following this rule. Otherwise, you might need to migrate it later.

Reply:
done.